govuk_navigation_helpers.gemspec

```diff
@@ -19,6 +19,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_runtime_dependency "gds-api-adapters", ">= 43.0"
+  spec.add_runtime_dependency "activesupport"
```

Comment:
That's 'true. However, this gem is only used in the context of Rails apps, so `activesupport` will already be present.